settings.js

```diff
@@ -27,7 +27,7 @@ window.spinnakerSettings = {
       },
       defaultSecurityGroups: ['nf-datacenter-vpc', 'nf-infrastructure-vpc', 'nf-datacenter', 'nf-infrastructure'],
       loadBalancers: {
-        // if true, VPC load balancers will be created as internal load balancers iff the selected subnet has a purpose
+        // if true, VPC load balancers will be created as internal load balancers if the selected subnet has a purpose
```

Comment:
this was intentional (iff == if and only if), but now that I'm seeing this, it was unnecessary and this makes sense without confusing the reader so I like it.